src/components/modals/RoleModal.vue

```diff
@@ -11,15 +8,41 @@
           : "bluffing"
       }}
     </h3>
-    <ul class="tokens">
-      <li
-        v-for="role in availableRoles"
-        :class="[role.team]"
-        :key="role.id"
-        @click="setRole(role)"
-      >
-        <Token :role="role" />
+    <ul>
```

Comment:
Why is this an `<ul>`? There doesn't seem to be a need to wrap the button-group `div` inside an unordered list.
I'd simply structure it
```
<ul .token>
 <li>...
</ul>
<div .button-group>
 <div .button>...
</div>
```